src/components/reservation/helper.ts

```diff
@@ -149,4 +150,21 @@ export const getAuthorLine = (
     : [t("materialHeaderAuthorByText"), author, year].join(" ");
 };
 
+export const getManifestationsToReserve = (
+  reservableManifestations: Manifestation[] | null,
+  isPeriodical?: boolean
+) => {
+  if (isPeriodical) {
+    return reservableManifestations;
+  }
+  if (!reservableManifestations || reservableManifestations.length < 1) {
+    return null;
+  }
+  // If the work isn't fictional we only want to reserve the latest manifestation.
+  if (!materialIsFiction(reservableManifestations[0])) {
+    return [getLatestManifestation(reservableManifestations)];
```

Comment:
The comment writes out what the code does in text. That is nice if the code is complex but more importantly is always *why* - especially if it refers to business logic.